src/components/navigation/Navigation.tsx

```diff
@@ -51,43 +49,23 @@ const Navigation = ({ onNavigationToggle, open, width }: Props) => {
                 <List aria-label="main application navigation">
                     <ListItemLink
                         icon={<ExploreIcon />}
-                        title={intl.formatMessage({
-                            id: routeDetails.home.title,
-                        })}
-                        link={routeDetails.home.path}
-                        key={intl.formatMessage({
-                            id: routeDetails.home.title,
-                        })}
+                        title={routeDetails.dashboard.title}
+                        link={routeDetails.dashboard.path}
```

Comment:
Key was left over from when this was made in a loop. Passing in the message id to have the component translate.